paprika/restraints/openmm.py

```diff
@@ -23,10 +28,10 @@ def apply_positional_restraints(coordinate_path: str, system, force_group: int =
         The system object to add the positional restraints to.
     force_group : int, optional
         The force group to add the positional restraints to.
+    kpos : float, optional
+        The force constant for restraining the dummy atoms (kcal/mol/Å^2).
```

Comment:
Is this the only place where the units *must* be specified as defined in the docstring?

Reply:
In most cases, we only need units when defining restraints with `static_DAT_restraint` and `DAT_restraint`. There are only a few modules that require units and are documented in the docstring.